### ISSUE TITLE
Not working on Samsung s6

### ISSUE THREAD

**Issue by TabraizHassan** [KieronQuinn/TapTap#31]
I have samsung s6 with android 7.0. I tried changing all settings like pixel 3, 4 and 4xl but still doesn't work.
Also, back of my smartphone is covered by glass protector, is it causing the issue? But i tapped really hard on the back of the device many times, still hasn't work even for one time🤷🏻‍♂️

**Comment by KieronQuinn** [comment]
Does it not work at all? Any crashes?

**Comment by tolgaaaltas** [comment]
> Does it not work at all? Any crashes?

On my OnePlus 5, everything works fine but on my little brother's Xiaomi Mi 5S doesn't work. There is no crash on Mi 5S.

**Comment by TabraizHassan** [comment]
> Does it not work at all? Any crashes?

In beta 0.1, at first it crashed only one time after i enabled it in accessiblity settings. But after that it never crashed. Well, to begin with it never worked ever. 
After installing new updates, still isn't working no matter what confifuration or settings I try🤷🏻‍♂️

**Comment by KieronQuinn** [comment]
Please try with the newer versions, see if it's still happening

**Comment by TabraizHassan** [comment]
Still not working(Beta 0.8), tried all settings. Also caused my device to
crash when an advanced setting 'Use split services' was enabled and device
was restarted. I wasnt able to even pass my lockscreen and screen kept
blacking out. Luckily, somehow I manged to restart the device again and
with some more difficulty was able to unistall the app, as I kept getting
the error message' Tap Tap has stopped working' repeatedly.
